modules/gitops-updating-the-rollout.adoc

```diff
@@ -0,0 +1,89 @@
+----
+spec:
+  replicas: 5
+  strategy:
+    canary: #<1>        
+      steps: #<2>
+      - setWeight: 20 #<3>       
+      - pause: {}  #<4>         
+  # (...)
+----
+<1> The deployment strategy that the rollout must use.
+<2> The steps for the rollout. This example gradually routes 20%, 40%, 60%, and 80% of traffic to the canary version.
+<3> The percentage of traffic that must be directed to the canary version. A value of 20 means that 20% of traffic is directed to the canary version.
+<4> Specification to the Argo Rollouts controller to pause indefinitely until it finds a request for promotion.
+
+. Watch the progression of your rollout by running the following command:
++
+[source,terminal]
+----
+$ oc argo rollouts get rollout rollouts-demo --watch -n <namespace> <1>
+----
+<1> Specify the namespace where the `Rollout` CR is defined.
++
+.Sample output
+[source,terminal]
+----
+Name:            rollouts-demo
+Namespace:       spring-petclinic
+Status:          ॥ Paused
+Message:         CanaryPauseStep
+Strategy:        Canary
+  Step:          1/8
+  SetWeight:     20
+  ActualWeight:  20
+Images:          argoproj/rollouts-demo:blue (stable)
+                 argoproj/rollouts-demo:yellow (canary)
+Replicas:
+  Desired:       5
+  Current:       5
+  Updated:       1
+  Ready:         5
+  Available:     5
+
+NAME                                       KIND        STATUS     AGE    INFO
+⟳ rollouts-demo                            Rollout     ॥ Paused   9m51s  
+├──# revision:2                                                          
+│  └──⧉ rollouts-demo-6cf78c66c5           ReplicaSet  ✔ Healthy  99s    canary
+│     └──□ rollouts-demo-6cf78c66c5-zrgd4  Pod         ✔ Running  98s    ready:1/1
+└──# revision:1                                                          
+   └──⧉ rollouts-demo-687d76d795           ReplicaSet  ✔ Healthy  9m51s  stable
+      ├──□ rollouts-demo-687d76d795-75k57  Pod         ✔ Running  9m50s  ready:1/1
+      ├──□ rollouts-demo-687d76d795-jsxg8  Pod         ✔ Running  9m50s  ready:1/1
+      ├──□ rollouts-demo-687d76d795-rsgtv  Pod         ✔ Running  9m50s  ready:1/1
+      └──□ rollouts-demo-687d76d795-xrmrj  Pod         ✔ Running  9m50s  ready:1/1
+----
++
+The rollout is now in a paused status, because there is no pause duration specified in the rollout's update strategy configuration.
+
+. Repeat the previous step to test the newly deployed version of application and ensure that it is working as expected. For example, verify the application by interacting with the application through the browser and try running tests or observing container logs.
++
+The Rollout will remain paused until you advance it to the next step. 
+
+After you have verified that the new version of the application is working as expected, you can decide whether to continue with promotion or to abort the rollout. Accordingly, follow the instructions mentioned in "Promoting the rollout" or "Manually aborting the rollout".
```

Comment:
```suggestion
After you verify that the new version of the application is working as expected, you can decide whether to continue with promotion or to abort the rollout. Accordingly, follow the instructions in "Promoting the rollout" or "Manually aborting the rollout".
```
If these are Additional resource links, say so. Then the reader knows where to look for this information.